stage1_fly/run/main.go

```diff
@@ -218,8 +218,16 @@ func stage1() int {
 		return 1
 	}
 
-	// TODO: insert environment from manifest
-	env := []string{"PATH=/bin:/sbin:/usr/bin:/usr/local/bin"}
+	defaultEnv := []string{"PATH=/bin:/sbin:/usr/bin:/usr/local/bin"}
+	envList := p.Manifest.Apps[0].App.Environment
+
+	env := make([]string, len(envList))
+	for _, e := range envList {
+		if _, exists := envList.Get(defaultEnv[0]); !exists {
```

Comment:
The function [Get](https://github.com/appc/spec/blob/master/schema/types/environment.go#L84) accepts only the name of the environment variable and not its content.

[environment.go](https://github.com/appc/spec/blob/master/schema/types/environment.go#L31):
```
type EnvironmentVariable struct {
	Name  string `json:"name"`
	Value string `json:"value"`
}
```

But here, the argument is "PATH=/bin:/sbin:/usr/bin:/usr/local/bin".